writers/influxdb/influxdb.go

```diff
@@ -41,10 +41,7 @@ func (repo *influxRepo) Save(msg mainflux.Message) error {
 	}
 
 	bp.AddPoint(pt)
-	if err := repo.client.Write(bp); err != nil {
-		return err
```

Comment:
Maybe add public error?

Reply:
We could. @mijicd Suggestions? Currently, I see no benefits of doing that, but I don't mind exporting an error, as well.

Reply:
There's no point of adding that.